test/runtime/cli.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2017 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package RuntimeTest
+
+import (
+	"fmt"
+
+	"github.com/cilium/cilium/test/helpers"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
+)
+
+var _ = Describe("RuntimeCLI", func() {
+
+	var initialized bool
+	var logger *logrus.Entry
+	var vm *helpers.SSHMeta
+
+	initialize := func() {
+		if initialized == true {
+			return
+		}
+		logger = log.WithFields(logrus.Fields{"testName": "RuntimeCLI"})
+		logger.Info("Starting")
+		vm = helpers.CreateNewRuntimeHelper(helpers.Runtime, logger)
+		vm.WaitUntilReady(100)
+		vm.NetworkCreate(helpers.CiliumDockerNetwork, "")
+
+		res := vm.SetPolicyEnforcement(helpers.PolicyEnforcementDefault)
+		res.ExpectSuccess()
+
+		areEndpointsReady := vm.WaitEndpointsReady()
+		Expect(areEndpointsReady).Should(BeTrue())
+
+		initialized = true
+	}
+
+	BeforeEach(func() {
+		initialize()
+	})
+
+	AfterEach(func() {
+		if CurrentGinkgoTestDescription().Failed {
+			vm.ReportFailed()
+		}
+	})
+
+	Context("Identity testing", func() {
+		It("Test labelsSHA256", func() {
+			By("Creating containers")
```

Comment:
This `By` Here doesn't make too much sense to me, there is only one test and no steps in the test, so I think that can be deleted. (Doesn't add a real value)

Reply:
Done.